src/main.js

```diff
@@ -19,6 +19,7 @@ import pullrequestCommitAmount from './pullrequest-commit-amount';
 import insertPullrequestTemplate from './pullrequest-template';
 import addSidebarCounters from './sidebar-counters';
 import syntaxHighlight from './syntax-highlight';
+import mergeCompare from './merge-compare';
```

Comment:
I'm not sure about this name :S, I have a tough time relating what the feature does to the name `merge-compare`, can you come up with something better?

Reply:
`compare-page-pull-request`?